src/ucp/core/ucp_ep.c

```diff
@@ -471,16 +473,30 @@ ucs_status_t ucp_ep_create_accept(ucp_worker_h worker,
         }
         break;
     case UCP_WIREUP_SOCKADDR_CD_CM_ADDR:
-        return UCS_ERR_NOT_IMPLEMENTED;
+        ucs_assert(ucp_worker_sockaddr_is_cm_proto(worker));
+        for (i = 0; i < remote_addr.address_count; ++i) {
+            remote_addr.address_list[i].dev_addr = conn_request->remote_dev_addr;
+        }
+        params.field_mask  |= UCP_EP_PARAM_FIELD_CONN_REQUEST;
+        params.conn_request = conn_request;
+        status = ucp_ep_cm_server_create_connected(worker, &params,
+                                                   &remote_addr, ep_p);
+        if (status != UCS_OK) {
+            goto out_free_address;
+        }
+
+        (*ep_p)->flags                 |= UCP_EP_FLAG_LISTENER;
+        ucp_ep_ext_gen(*ep_p)->listener = conn_request->listener;
+        break;
     default:
-        ucs_fatal("client data contains invalid address mode %d",
+        ucs_fatal("sockaddr data contains invalid address mode %d",
```

Comment:
isn't this function valid only for the server side?

Reply:
yes